lightyear/src/transport/webtransport/server.rs

```diff
@@ -58,7 +58,7 @@ impl ServerTransportBuilder for WebTransportServerSocketBuilder {
 
         let mut config = ServerConfig::builder()
             .with_bind_address(self.server_addr)
-            .with_identity(&self.certificate)
+            .with_identity(self.certificate)
```

Comment:
why was this changed?

Reply:
wtransport version bump in recent commit changed the api